week-2-wide-perspective/w02e03-pagination-problem/src/middlewares/paginate.middleware.js

```diff
@@ -0,0 +1,11 @@
+export const paginate = function (req, res, next) {
```

Comment:
generalnie to jest zrobione dobrze i logika ok, fajnie zgrupowane w `.paginate`

A może dorzucisz jeszcze funkcjonalność samodzielnego wyboru nazw pól w `.query` ?